src/components/menu/RewardMenu.sc.tsx

```diff
@@ -1,14 +1,21 @@
 import { brandColors } from '@giveth/ui-design-system';
 import styled from 'styled-components';
 
-export const RewardMenuContainer = styled.div`
+interface isMounted {
+	isMounted: boolean;
+}
+
+export const RewardMenuContainer = styled.div<isMounted>`
 	position: absolute;
 	top: 40%;
 	right: 0;
 	width: 260px;
 	background-color: ${brandColors.giv[900]};
 	border: 1px solid ${brandColors.giv[600]};
 	border-radius: 10px 0 10px 10px;
-	padding: 38px 24px;
+	padding: ${props => (props.isMounted ? '38px 12px' : '12px')};
```

Comment:
Is this props (isMounted) needed?

Reply:
@aminlatifi 
yes, we need it for the opening animation.